src/tools/ceph_objectstore_tool.cc

```diff
@@ -3551,6 +3551,38 @@ int main(int argc, char **argv)
 	fs->apply_transaction(osr, std::move(*t));
       }
       cout << "Marking complete succeeded" << std::endl;
+    } else if (op == "reset-last-complete") {
+      if (!force) {
+        std::cerr << "WARNING: reset-last-complete is extremely dangerous and almost "
+                  << "certain to lead to permanent data loss unless you know exactly "
+                  << "what you are doing. Pass --force to proceed anyway."
+                  << std::endl;
+        ret = -EINVAL;
+        goto out;
+      }
+      ObjectStore::Transaction tran;
+      ObjectStore::Transaction *t = &tran;
+
+      if (struct_ver < PG::compat_struct_v) {
+        cerr << "Can't reset-last-complete, version mismatch " << (int)struct_ver
```

Comment:
@Liuchang0812 I believe It would be good to replace "Can't" with "Can not".
It also makes it easier to translate.